sdk/python/requirements.txt

```diff
@@ -1,4 +1,5 @@
 certifi >= 14.05.14
+future; python_version<="2.7"
```

Comment:
Just to clarify, is this for backward compatibility with python 2.7 and earlier?

Reply:
I may have removed it in my refactor of python sdk but this regenerated it so it may be adding this in as part of the sdk generation.

Reply:
I did not manually add that in.